dtos/messages/query-message.dto.ts

```diff
@@ -1,18 +1,16 @@
 import { ApiPropertyOptional } from '@nestjs/swagger';
-import { IsNumberString, IsOptional } from 'class-validator';
+import { IsOptional } from 'class-validator';
 
 export class QueryMessageDto {
   @ApiPropertyOptional({
     description: 'Channel Id',
   })
   @IsOptional()
-  @IsNumberString()
   channel?: string;
 
   @ApiPropertyOptional({
     description: 'Sender Id',
   })
   @IsOptional()
-  @IsNumberString()
```

Comment:
I am not sure of the reason of this change. It is information for 'class-validator'.